lib/devise_security_extension/models/password_expirable.rb

```diff
@@ -51,6 +51,12 @@ def update_password_changed
           self.password_changed_at = Time.now if (self.new_record? or self.encrypted_password_changed?) and not self.password_changed_at_changed?
         end
 
+        def expired_password_after_numeric?
+          return @_numeric if defined?(@_numeric)
```

Comment:
Is it actually a good idea to memoize this? Could there be a scenario where this gets checked and then the column is made to be `nil`?

```ruby
expired_password_after_numeric? #=> true
self.expire_password_after = nil
expired_password_after_numeric? #=> true
```

Reply:
Unfortunately I don't know the code well enough to say.